src/Oscar.jl

```diff
@@ -84,6 +84,7 @@ function __init__()
         (GAP.Globals.IsMatrixGroup, MatrixGroup),
         (GAP.Globals.IsFpGroup, FPGroup),
     ])
+    GAP.Packages.load("forms")
```

Comment:
Is there a reason why this command must be executed at runtime (in `__init__`)?

Reply:
When I wrote the code for the first time, something was not working putting the instruction `GAP.Packages.load("forms")` in the file `forms.jl`. But now, I moved it and it seems work perfectly. Maybe something changed in the meantime. If it works, would you suggest to load the GAP package in the file `forms.jl` (or somewhere else)?

Reply:
Sorry, problem on my side. This statement **must** be executed at runtime because it has to be executed in the GAP that is currently running.